core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -948,10 +948,8 @@ private void deleteSubjectCompatibility(String subject) throws StoreException {
     this.kafkaStore.delete(configKey);
   }
 
-  /**
-   * For testing.
-   */
-  KafkaStore<SchemaRegistryKey, SchemaRegistryValue> getKafkaStore() {
+
+  public KafkaStore<SchemaRegistryKey, SchemaRegistryValue> getKafkaStore() {
```

Comment:
Ah, also need to review KafkaStore public APIs since this is making that interface public.